### DISCUSSION TITLE
Add examples of I2C expander addresses

### DISCUSSION THREAD

**Discussion by mDubovsky** [pilotak/eth-io#15] [Ideas]
It would be beneficial to have I2C addresses visualized in a way where I don't have to look at eagle files or expander datasheets during assembly. Like a cheat-sheet for every module. Something that would look like this, but of course not MS Paint graphic like this one. This is just for reference. I would definitely add that to the wiki (assembly section) and maybe even pdf in the repository for quick reference.
![image](https://user-images.githubusercontent.com/9304179/208685292-f4d38085-dd92-4f56-aba2-d3dc553ad449.png)

**Comment by petrzoufaly** [comment]
![i2c_address_example](https://user-images.githubusercontent.com/59135977/208686015-b7e42b5d-c5bc-4838-adb4-fcc98a234d3e.png)
not sure if there is a room on the board somewhere, this is what I did on similar board

**Reply by pilotak** [reply]
the only space left is here 
![addr](https://user-images.githubusercontent.com/8101753/208703463-11f0866d-4414-4def-bec2-2f395f5fbc10.png)

**Reply by mDubovsky** [reply]
At this point I think it would be more handy to have it on the screen than on the other side of the PCB. With @petrzoufaly example you can at least see the table while you're soldering. It kind of misses the point if it's on the other side of the board.

**Reply by petrzoufaly** [reply]
Better than nowhere, but I agree, probably in wiki is good enough, you're not changing this every 10 minutes, right

**Reply by pilotak** [reply]
Wiki should be enough. Most of the time it will be one time configuration falling to the default

**Comment by petrzoufaly** [comment]
Maybe that's a good place for some basic electric parameters of the board too, those you mention on the sticker

**Reply by mDubovsky** [reply]
Not a huge fan of this approach. It's pretty common to have things like "24V" right next to the terminal where the voltage is applied but I'm not so sure about putting whole text paragraphs / tables on the PCB.

**Reply by pilotak** [reply]
I have to agree with @mDubovsky. The board is in the enclosure with a stickers on. There is everything you need to know.
